src/program/snabbnfv/traffic/traffic.lua

```diff
@@ -41,6 +41,11 @@ function traffic (pciaddr, confpath, sockpath)
          local mtime2 = C.stat_mtime(confpath)
          if mtime2 ~= mtime then
             print("Loading " .. confpath)
+            if mtime2 == 0 then
```

Comment:
> Can we skip this change or make it in another way that does not change the high-level behavior of the traffic process?

what I wanted to avoid was the forcefully killing of the process, so that the profiling output could be seen.  maybe it would be better to catch `SIGTERM` and do an ordered exit instead of just dying